src/utils/getUserById.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/User';
+import usersFromServer from './../api/users';
```

Comment:
[CODE STYLE] - Avoid using relative paths that go up and down the directory structure. It's better to use absolute paths or configure your module resolution to make imports cleaner and more manageable. For instance, if you have a `baseUrl` configured in your `tsconfig.json`, you could import like this: `import usersFromServer from 'api/users';`